src/configs/V1CNC_SkrPro_DualLR_2209_WiFi

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
I'd like this renamed to V1CNC_SkrPro_DualLR_2209_ESP3D. Anyone who knows will recognize it, and we won't have people wondering about wifi when they don't understand the consequences.

Reply:
There's no reason why we can't also add in the non low rider version(s), right?

It's a shame these aren't more reusable. But I also love how clean and direct they are. So I am excusing us from the DRY rule.

Reply:
Since this isn't added into the workflows, this won't be automatically built, and it won't be in an releases until we add those files.

Reply:
Good points. I will rename the file. Do you want me to add the ESP3D accessory file and create configs for each of the SkrPro variants?

Reply:
At most, adding it to the serial and dual (non-lr) versions would be ok. This isn't applicable to the other boards, and it is one line, so I don't think making a ESP3D accessory makes it any better.